pkgs/development/python-modules/python-watcherclient/default.nix

```diff
@@ -19,7 +19,7 @@
 
 buildPythonPackage rec {
   pname = "python-watcherclient";
-  version = "4.4.0";
+  version = "4.5.0";
```

Comment:
Note for reviewers: no change for this version

See https://opendev.org/openstack/python-watcherclient/compare/4.4.0...4.5.0

Reply:
see https://github.com/openstack/python-watcherclient/commit/56e47c38e209010bbd9f3250ee959efaa47e61dd